compute-labs/aws/aws-lambda-function-with-http-trigger.md

```diff
@@ -40,7 +40,8 @@ In this exercise, you will create an AWS Lambda Function for your serverless app
 6. On the same screen, fill in the following information:
    - *Function name* → `[initials]-awslbdhttplab-fun`
    - *Runtime* → `Node.js 14.x`
-7. Click on the ![Create function button](media/aws-create-function-orange-button.png) button.
+7. Change the default execution role to `Use an existing role`. For the existing role select `Lambda_Role`.
```

Comment:
`Lambda_Role` is not a standard role. I will need to look for more appropriate role that is built-in and can be used for that purpose.